src/styles/components/_code-pattern.scss

```diff
@@ -0,0 +1,37 @@
+@import '../settings/colors';
+
+// =============================================================================
+// CODE PATTERN component
+//
+// A component that displays the code and a demo side by side.
+//
+// =============================================================================
+
+.code-pattern {
+  width: 100%;
+
+  .code-pattern__content {
+    display: grid;
+    gap: 1rem; // sass-lint:disable-line no-misspelled-properties
+    grid-template-columns: 50% auto;
+    justify-items: stretch;
+    height: 300px;
+    margin-bottom: 1rem;
+    overflow: scroll;
```

Comment:
You can remove this. `overflow:scroll` creates visible gutters on macOS even if there is no overflow.

![image](https://user-images.githubusercontent.com/1066253/128763352-279bc4e4-ff83-4c9d-8972-d6ba51e3e0e5.png)

You can use `overflow: auto` to make them go away. But in this case I don't think you want this element to have any overflow, it seems like you'd probably want the actual demo and the tabs to have the overflow.